fh-messaging/lib/mongo_functions/mbaas_reduce_apprequestsdest.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable*/
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function mbaas_reduce_apprequestsdest(k, mbaasMetrics) {
+  // Take the domain and ts in the first vals entry as the reduced elements domain and ts
+  // var result = {};
+
+
+  // function forEach(iterable, cb) {
+  //   for (var i in iterable) {
+  //     if (iterable.hasOwnProperty(i)) {
+  //       cb(iterable[i], i);
+  //     }
+  //   }
+  // }
+
+  // forEach(mbaasMetrics, function(appMetrics) {
+  //   forEach(appMetrics, function(metrics, p) {
+  //     if (typeof result[p] !== 'undefined') {
+  //       result[p] += metrics;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)